conan/api/output.py

```diff
@@ -82,24 +82,25 @@ def is_terminal(self):
         return hasattr(self.stream, "isatty") and self.stream.isatty()
 
     def writeln(self, data, fg=None, bg=None):
-        self.write(data, fg, bg, newline=True)
+        return self.write(data, fg, bg, newline=True)
 
     def write(self, data, fg=None, bg=None, newline=False):
         if conan_output_level > LEVEL_NOTICE:
-            return
+            return self
         if self._color and (fg or bg):
             data = "%s%s%s%s" % (fg or '', bg or '', data, Style.RESET_ALL)
 
         if newline:
             data = "%s\n" % data
         self.stream.write(data)
         self.stream.flush()
+        return self
 
     def rewrite_line(self, line):
         tmp_color = self._color
         self._color = False
         TOTAL_SIZE = 70
-        LIMIT_SIZE = 32  # Hard coded instead of TOTAL_SIZE/2-3 that fails in Py3 float division
+        LIMIT_SIZE = TOTAL_SIZE // 2 - 3
```

Comment:
The old comment complained about Py3 division being float by default. Using the built-in int division fixes the issue (`//`)